doc/user-stories/expert-partitioner.md

```diff
@@ -0,0 +1,462 @@
+
+# Expert partitioner specs
+
+## When 'Hard Disks' is selected in the tree view
+  * shows a table with the columns: Device, Size, F, Enc, Type, FS Type, Label, Mount Point, Start, End
```

Comment:
and in fact old expert partitioner have here configurable columns, try change columns in options. I am just not sure if we want it also in new storage-ng as it is quite hidden and not much intuitive.